arbnode/node.go

```diff
@@ -671,6 +671,7 @@ func createNodeImpl(
 			dataAvailabilityService = das.NewReadLimitedDataAvailabilityService(dataAvailabilityService)
 		}
 	}
+	dataAvailabilityService = das.NewTimeoutWrapper(dataAvailabilityService, config.DataAvailability.RequestTimeout)
```

Comment:
This needs to go in the above `if` statement, there are a number of places we check for absence of the DAS by checking if the interface is nil.

Reply:
Done.